admin/commands/management/add_postgresql.go

```diff
@@ -61,24 +61,25 @@ type AddPostgreSQLCommand struct {
-	DisableQueryExamples bool              `name:"disable-queryexamples" help:"Disable collection of query examples"`
-	MetricsMode          string            `enum:"${metricsModesEnum}" default:"auto" help:"Metrics flow mode, can be push - agent will push metrics, pull - server scrape metrics from agent or auto - chosen by server"`
-	DisableCollectors    []string          `help:"Comma-separated list of collector names to exclude from exporter"`
-	ExposeExporter       bool              `name:"expose-exporter" help:"Optionally expose the address of the exporter publicly on 0.0.0.0"`
-	AutoDiscoveryLimit   int32             `default:"0" placeholder:"NUMBER" help:"Auto-discovery will be disabled if there are more than that number of databases (default: server-defined, -1: always disabled)"`
+	QuerySource            string            `default:"pgstatmonitor" help:"Source of SQL queries, one of: pgstatements, pgstatmonitor, none (default: pgstatmonitor)"`
+	Environment            string            `help:"Environment name"`
+	Cluster                string            `help:"Cluster name"`
+	ReplicationSet         string            `help:"Replication set name"`
+	CustomLabels           map[string]string `mapsep:"," help:"Custom user-assigned labels"`
+	SkipConnectionCheck    bool              `help:"Skip connection check"`
+	CommentsParsing        string            `enum:"on,off" default:"off" help:"Enable/disable parsing comments from queries. One of: [on, off]"`
+	TLS                    bool              `help:"Use TLS to connect to the database"`
+	TLSCAFile              string            `name:"tls-ca-file" help:"TLS CA certificate file"`
+	TLSCertFile            string            `help:"TLS certificate file"`
+	TLSKeyFile             string            `help:"TLS certificate key file"`
+	TLSSkipVerify          bool              `help:"Skip TLS certificates validation"`
+	MaxQueryLength         int32             `placeholder:"NUMBER" help:"Limit query length in QAN (default: server-defined; -1: no limit)"`
+	DisableQueryExamples   bool              `name:"disable-queryexamples" help:"Disable collection of query examples"`
+	MetricsMode            string            `enum:"${metricsModesEnum}" default:"auto" help:"Metrics flow mode, can be push - agent will push metrics, pull - server scrape metrics from agent or auto - chosen by server"`
+	DisableCollectors      []string          `help:"Comma-separated list of collector names to exclude from exporter"`
+	ExposeExporter         bool              `name:"expose-exporter" help:"Optionally expose the address of the exporter publicly on 0.0.0.0"`
+	AutoDiscoveryLimit     int32             `placeholder:"NUMBER" help:"Auto-discovery will be disabled if there are more than that number of databases (default: server-defined, -1: always disabled)"`
+	MaxExporterConnections int32             `placeholder:"NUMBER" help:"Maximum number of connections to the exporter (default: server-defined)"`
```

Comment:
as far as I see it's not really server-defined, but exporter defined.

Reply:
```suggestion
	MaxExporterConnections int32             `placeholder:"NUMBER" help:"Maximum number of connections that exporter can open at the same time (default: exporter-defined)"`
```

Like this?

Reply:
Fixed description, but kept `server-defined`. Please let me know if you want to change it to something else.